docs/elements.md

```diff
@@ -1194,6 +1194,7 @@ Attribute | Type | Default | Description
 `directory` | text | "clientFilesQuestion" | The directory that contains the file, either 'clientFilesQuestion' or 'clientFilesCourse' (see [client and server files](clientServerFiles.md)). A directory cannot be specified if `type='dynamic'`.
 `width` | number | `None` | Width of image (e.g., '250px').
 `inline` | boolean | false | Display figure inline with text (true) or on a separate line (false).
+`alt-text` | text | "PrairieLearn Figure" | Provide alt (alternative) text to improve accessibility of figures by describing the image or the purpose of the image.
```

Comment:
```suggestion
`alt-text` | text | "PrairieLearn Figure" | Provide alt (alternative) text to improve accessibility of figures by describing the image or the purpose of the image. For basic guidelines on creating good alt text, see [WebAIM's article on alternative text](https://webaim.org/techniques/alttext/).
```
Maybe it's worth linking to some basic alt text guidelines? Just a thought.

Reply:
Can you update the example course questions with appropriate alt texts? At least `exampleCourse/questions/element/figure`.

Reply:
add alt text to all figures in `exampleCourse/questions/element/figure`

Reply:
> Couldn't use alt or alt-text as a variable name in Python because alt is a reserved word, so apologies on the inconsistent naming of the variables.

I understand this, but I think `alt` would be a better name for the attribute so that it matches the underlying web platform. I don't think that would cause an issue with Python reserved words.

Reply:
Also, I can't find any documentation about `alt` being a reserved word - could you point me to that?

Reply:
Good point. I agree that we should use `alt` for the attribute name, but `alt_text` for the Python variable.

Reply:
Also, I think the default value needs to be changed to `""` in the docs above.

Reply:
hmmm...I'm not getting the error with alt in Python that I was getting earlier. It refused to work for me to until I changed alt to alt_text. It's behaving now with alt or alt_text. Leaving html/mustache as alt and leaving Python as alt_text for now. Updated the examples in `exampleCourse/questions/element/figure`